docs/content/guides/getting-started/saving-data.md

```diff
@@ -261,7 +256,7 @@ const ExampleComponent = () => {
       </div>
 
       <output className="console" id="output">{output}</output>
-    </Fragment>
+    </>
```

Comment:
Thank you, I guess it was a lot of work, but I think that this example was translated too literally into React. Let's keep it for now. I will make a new issue to rewrite this example in both JS and React.

Reply:
To clarify, I mean that the outdated `ajax` function can be replaced by `fetch` in the JS and React examples. Maybe this can be done in this PR after all.

Reply:
Will rewrite

Reply:
Looking good, thanks!